data/assets/scene/digitaluniverse/exoplanets_candidates.asset

```diff
@@ -19,13 +19,19 @@ local Object = {
     Type = "RenderablePointCloud",
     Enabled = false,
     Opacity = 0.99,
-    File = speck .. "exoplanet_candidates.speck",
+    File = speck .. "expl_candidates.speck",
```

Comment:
I think the old name was easier to understand, and not that much longer.

Reply:
@brianpatrick Alex and I talked and we leave the naming of the files entirely up to you. Don't feel the need to change it (especially at this stage, where I have already done some of the work for uploading the files to our data server)